src/test/java/pl/edu/icm/pl/mxrdr/extension/importer/pdb/PdbApiCallerTest.java

```diff
@@ -0,0 +1,71 @@
+package pl.edu.icm.pl.mxrdr.extension.importer.pdb;
+
+import kong.unirest.GetRequest;
+import kong.unirest.HttpResponse;
+import kong.unirest.UnirestInstance;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class PdbApiCallerTest {
```

Comment:
Mockito is not a good choice for testing API clients. I'd go with WireMock instead to test if we construct the HTTP request correctly and if we can parse HTTP response correctly. Both HTTP request and response are the I/O of the API client and it's worth to test this I/O contract to the extent it's used in our code.